src/widgets/ot_nodeready.eliom

```diff
@@ -27,12 +27,12 @@ let rec node_in_document node =
 let age = ref 0
 let watched = ref []
 
-(* We remove watched elements after two unload events to avoid memory
+(* We remove watched elements after two undead events to avoid memory
    leaks due to elements that are never inserted in the page. *)
 let active = ref false
 let remove_from_dead_pages () =
   let rec loop () =
-    Eliom_client.onunload @@ fun () ->
+    Eliom_client.Page_status.ondead @@ fun () ->
```

Comment:
Are you sure it works to call repeatedly `Eliom_client.Page_status.ondead` in a loop? Which page are we referring to?